_layouts/search_packages.html

```diff
@@ -75,6 +75,9 @@ <h2>Active Distributions</h2>
       <p>
         <a href="https://docs.ros.org/en/jazzy/Releases/Release-Jazzy-Jalisco.html">ROS 2 Jazzy</a>
       </p>
+      <p>
+        <a href="https://docs.ros.org/en/jazzy/Releases/Release-Kilted-Kaiju.html">ROS 2 Kilted</a>
```

Comment:
Just to match the other links:

```suggestion
        <a href="https://docs.ros.org/en/kilted/Releases/Release-Kilted-Kaiju.html">ROS 2 Kilted</a>
```

Reply:
Nice catch, I guess I was a little too quick to toss this off!

Committed in my branch.